### PR TITLE
Fixed : sending newline characters

### PR DESCRIPTION
**Pull request by viveksb007**
now trims white spaces and newline characters.
Removed Relative layout default padding as it was looking unusual with android default pull animation

### CODE REVIEW

**Comment by the-dagger** [comment]
Which issue is this Pull Request for?

**Comment by viveksb007** [comment]
For #57

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -10,7 +10,6 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:orientation="vertical"
-        android:paddingTop="@dimen/activity_vertical_margin"
```
Comment (gogeta95) [inline app/src/main/res/layout/activity_main.xml]: why this was done?

Comment (viveksb007) [inline app/src/main/res/layout/activity_main.xml]: that padding was looking unusual with android default pull animation

Comment (gogeta95) [inline app/src/main/res/layout/activity_main.xml]: you should mention that in the decription whatevr changes you make, please follow best practices.

Comment (viveksb007) [inline app/src/main/res/layout/activity_main.xml]: Done

**Comment by the-dagger** [comment]
Looks good

**Pull request merged**